pkg/assume/assume.go

```diff
@@ -246,8 +246,14 @@ func AssumeCommand(c *cli.Context) error {
 		// DO NOT REMOVE, this interacts with the shell script that wraps the assume command, the shell script is what configures your shell environment vars
 		// to export more environment variables, add then in the assume and assume.fish scripts then append them to this output preparation function
 		// the shell script treats "None" as an emprty string and will not set a value for that positional output
-		output := PrepareStringsForShellScript([]string{creds.AccessKeyID, creds.SecretAccessKey, creds.SessionToken, profile.Name, region, sessionExpiration})
-		fmt.Printf("GrantedAssume %s %s %s %s %s %s", output...)
+		if c.Bool("sso") {
```

Comment:
Instead of this,

You can use an empty string for profile.Name if it is sso then it will be set to "None" which in turn does not export the env var